lib/rest-connector.js

```diff
@@ -133,10 +135,10 @@ exports.initialize = function initializeDataSource(dataSource, callback) {
 exports.RestConnector = RestConnector;
 
 /**
- * The RestConnector constructor
+ * @class
+ * @classdesc The RestConnector constructor
```

Comment:
`@constructor` isn't meant to _replace_ class; it's just to differentiate between the two.
Again, it's not required.

The change I _do_ think is required is editing that description to be useful.